src/store/getters.js

```diff
@@ -89,12 +93,13 @@ export default {
   },
   swapProvider (state) {
     return (network, providerId) => {
-      const cacheKey = [network, providerId]
+      const supportedProviderId = mapLegacyProvidersToSupported[providerId] ? mapLegacyProvidersToSupported[providerId] : providerId
```

Comment:
Why is this needed now?

Reply:
Yes, they are doing the same and V3 will be deprecated soon . 

I need this line, because we have swaps in the history that have been using V3 and they are in the local storage. 
If we do not have a backward support this will result in visualization issues in the old swaps. 

I can also leave the old oneinchV3 and supported v3 and v4, but it does not make much sense to me